### ISSUE TITLE
u64 args doesn't work

### ISSUE THREAD

**Issue by bestouff** [killercup/quicli#40]
If I put this in the `Cli` struct:
```rust
    #[structopt(long = "timeout", short = "t", default_value = "3")]
    timeout: u64,
```
then `args.timeout`'s value is `0` by default, and `1` if I pass a `--timeout 5` argument.
If I replace `u64` with `u32` it works as expected.

**Comment by killercup** [comment]
This is structopt's behavior, so if you think this is a problem, please open an issue at https://github.com/TeXitoi/structopt :)

I _think_ this is by design, though. https://docs.rs/structopt-derive/0.1.6/structopt_derive/ says `u64` maps to "number of times the flag is used".

**Comment by bestouff** [comment]
Yep, found the corresponding issue: TeXitoi/structopt#30 - apprently it'll be fixed in `structopt` 0.2
